src/components/Cards/Card.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import styled from "styled-components";
+
+
+
+export interface CardProps {
+  title: string;
+  img: string;
+  alt?: string;
+  body?: string;
+  icon?: string;
+  price?: number;
+  restaraunt?: string;
+  isDish?: string;
+  isRest?: string;
+}
+
+const Card: React.FC<CardProps> = ({
+  title,
+  img,
+  body,
+  icon,
+  price,
+  alt,
+  restaraunt,
+  isDish,
+  isRest,
+}) => {
+  return (
+    <RestarauntContainer>
+      {isDish && (
+        <DishImage>
+          <img src={img} alt={alt}></img>
+        </DishImage>
+      )}
```

Comment:
Note that here you might get 0 as the output if you are not careful, not a mistake by any chance but I prefer to avoid a chance of this happening by using the ternary operator when I need to conditionally render.
Reference for logical && - https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
Good catch! Thanks!